src/logic/CanvasTouchToMouseAdapter.js

```diff
@@ -13,17 +13,19 @@ class CanvasTouchToMouseAdapter extends TouchToMouseAdapter {
   handleTouchMove(event) {
     this.updateActiveTouches(event)
 
-    if (event.touches.length === 2 && Object.keys(this.touches).length === 2) {
+    if ( Object.keys(this.touches).length === 2) {
       // Two-finger touch move
       this.handleTwoFingerGesture(event)
     }
-
-    this.forwardTouches(event)
+    else {
```

Comment:
Code style: Else should start on the same line as the closing brace (`} else {`)